src/mono/mono/mini/interp/transform.c

```diff
@@ -281,11 +281,14 @@ realloc_stack (TransformData *td)
 }
 
 static int
-get_stack_size (StackInfo *sp, int count)
+get_stack_size (TransformData *td, StackInfo *sp, int count)
```

Comment:
Nit: As the function already receives `td`, `sp` is redundant as a parameter.

Reply:
`get_stack_size` doesn't always receive `td->sp` as an argument here